handler.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports.hello = async (event) => {
```

Comment:
It would not be necessary to specify `async` in the function since there are no promises to wait, and also since we only return a value then we can do it immediately after `=>`
```suggestion
const hello = (event) => ({
    statusCode: 200,
    body: JSON.stringify(
      {
        message: 'Go Serverless v1.0! Your function executed successfully!',
        input: event,
      },
      null,
      2,
    ),
  });
```